### PR TITLE
Forms Automated Test - LPS-133781| master

### PR DESCRIPTION
**Pull request by MoryAlvesBR**

### CODE REVIEW

**Comment by wesleyroberts** [comment]
The macro's FormPortletConfiguration.changeLanguageTo and FormPortletConfiguration.viewTranslationLanguage , need changes.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro [hunk 1]
```diff
@@ -31,4 +35,10 @@ definition {
 			value1 = "Displaying Form: ${formName} (Modified)");
 	}
 
+	macro viewTranslationLanguage {
+		AssertElementPresent(
+			changeLanguageTo = "${symbolName}",
+			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
+	}
```
Comment (wesleyroberts) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro:42]: Change ${symbolName} for ${language}.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro [hunk 2]
```diff
@@ -2,6 +2,10 @@ definition {
 
 	var formName = "Created Form Name";
 
+	macro changeLanguageTo {
```
Comment (wesleyroberts) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro:5]: Change this macro for macro
 changeLanguageTo {
		Click(
			changeLanguageTo = "${getLanguage}",
			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
		Click(
			changeLanguageTo = "${changeLanguageTo}",
			locator1 = "FormPortletConfiguration#SELECT_LANGUAGE");
	}

**Comment by natocesarrego** [comment]
@wesleyroberts and @MoryAlvesBR, new changes were requested on https://github.com/liferay-forms/liferay-portal/pull/558, please apply them here.

Thanks.

**Comment by natocesarrego** [comment]
@MoryAlvesBR, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase [hunk 3]
```diff
@@ -427,14 +427,44 @@ definition {
...
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
 
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		Form.editElementSetFieldLabel(fieldValue = "Texto");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase:449]: @MoryAlvesBR, you should use the macro to change the Field Label and not the Element Set Field Label, can you please fix it?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@MoryAlvesBR, please run SF locally, because this pull have SF issues. Moreover, please put together all you changes in a single commit.

Thanks

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**